templates/metasploitMacOSBuilder.json

```diff
@@ -9,5 +9,5 @@
   "vagrantfile_template": "vagrant/vagrantfile-metasploitMacOSBuilder.tpl",
   "virtualbox_guest_os_type": "MacOS1015_64",
   "vmware_guest_os_type": "darwin18-64",
-  "version": "1.0.6"
+  "version": "1.0.7"
```

Comment:
Technically I haven't changed anything here, but I assumed lock-stepping the versions would be easiest for traceability

Reply:
Yes bump for any change at patch level is good for tracking build that was used.